src/exercises/Exercises.js

```diff
@@ -47,11 +49,15 @@ export default function Exercises({
   const [activeSessionDuration, clockActive, setActivityOver] =
     useActivityTimer();
 
+  useEffect(() => {
+    setNumberOfBookmarksToPractice(getNumberOfBookmarksToPractice(exerciseSequenceType));
+  }, [exerciseSequenceType]);
+
   function getExerciseSequenceType() {
-    let exerciseTypesList = DEFAULT_SEQUENCE;
```

Comment:
This is where we should include a line to "select" your feature.

My exercise is not currently in the code base but here an example from the file history:

```
import Feature from "../features/Feature";

if (Feature.tiago_exercises()) {
            exerciseTypesList = EXERCISE_TYPES_TIAGO;
        }
```